src/go/wsl-helper/pkg/dockerproxy/mungers/containers_create_windows.go

```diff
@@ -84,7 +84,7 @@ func mungeContainersCreate(req *http.Request, contextValue *dockerproxy.RequestC
 		}
 		logrus.WithFields(logrus.Fields{
 			"mount":     mount,
-			"traslated": translated,
+			"translate": translated,
```

Comment:
??

Reply:
It's a logging field; it should be
```suggestion
			"translated": translated,
```